src/components/Select/Select.tsx

```diff
@@ -96,6 +106,8 @@ export const Select = React.forwardRef<HTMLButtonElement, SelectProps>(function
         onClose,
         onOpenChange,
     });
+    const uniqId = useUniqId();
+    const selectId = id ?? uniqId;
```

Comment:
Maybe smth like this?
```js
const selectId = id || useUniqId();
```
Remove extra line and change operator case of empty string does not a valid id

Reply:
useUniqId is a hook and cannot be called conditionally.